src/components/scenes/SendScene.js

```diff
@@ -235,6 +240,19 @@ class SendComponent extends React.PureComponent<Props, State> {
     })
   }
 
+  handleFocusPin = () => {
+    if (this.pinInput && this.pinInput.current) {
+      this.pinInput.current.focus()
+    }
+  }
+
+  handleChangePin = (pin: string) => {
+    this.props.onChangePin(pin)
+    if (pin.length >= 4 && this.pinInput) {
```

Comment:
Don't use unnamed numbers in the code (like this 4). 
Create a constant and and use that. Also pass it in as 'maxLength' for the 'TextInput' component